backend/django/core/views/api_annotate.py

```diff
@@ -685,6 +694,8 @@ def label_admin_label(request, data_pk):
         if datum.irr_ind:
             Data.objects.filter(pk=datum.pk).update(irr_ind=False)
 
+        AdjudicateDescription.objects.filter(data_id=data_pk).update(isResolved=True)
```

Comment:
probably need to add this to the discard data call as well, so it's not hanging around if the data gets restored and put back in circulation